src/main/java/org/isf/dicom/manager/SourceFiles.java

```diff
@@ -209,8 +209,8 @@ else if (isDicom) {
 	 * @param patient
 	 * @throws Exception
 	 */
-	@SuppressWarnings("unused")
-	public synchronized static void loadDicom(FileDicom dicomFileDetail, File sourceFile, int patient) throws Exception {
+    @SuppressWarnings("unused")
+    public static synchronized void loadDicom(FileDicom dicomFileDetail, File sourceFile, int patient) throws Exception {
```

Comment:
could please you use the https://github.com/informatici/openhospital-core/blob/develop/.editorconfig included in the project?

Reply:
See above.